contents/security/Https.md

```diff
@@ -30,7 +30,7 @@ Author: [Boxin][1]
 HTTPS is the end-to-end encryption on data on top of HTTP to prevent network sniffing (eavesdropping data packets). In this tutorial, we will cover four questions to have a better understanding of https. The questions are:
 
 - [Why do we need HTTPS](#why-do-we-need-https)
-- [Why HTTPS is secure](#why-https-is-secure)
+- [Why HTTPS is secure](#why-is-https-secure)
```

Comment:
This section needs to follow title case too, then I think? Not sure though. You can take a call.

Reply:
I've decided to keep it non title-cased since the sentence above references them as "questions to be answered" rather than chapters. I have however added question marks to them and also the relevant ToC portions.